code/game/objects/items/dehy_carp.dm

```diff
@@ -36,7 +37,7 @@
 	//Wait for animation to end
 	sleep(6)
 	//Make space carp
-	var/mob/living/simple_animal/hostile/carp/C = new /mob/living/simple_animal/hostile/carp(get_turf(src))
+	var/mobtype/C = new mobtype(get_turf(src))
```

Comment:
oh yeah huh
i guess you would just remove the var definition and just have "new mobtype..." stuff, then replace mentions of the var C with "mobtype"